python/neuroglancer/pipeline/threaded_queue.py

```diff
@@ -148,6 +149,8 @@ def _consume_queue_execution(self, fn):
     """
     try:
       fn()
+    except Exception as err:
```

Comment:
Trying to decide if we should raise here and kill the thread. The major exception handling happens in `wait`. What do you think Nacho?